docs/home/howto/DATABASES.md

```diff
@@ -498,7 +497,7 @@ DROP EXTENSION IF EXISTS pgaudit;
 
 #### Note
 
-This feature is deprecated and will be removed.
+Native PostgreSQL replication is deprecated and removed.
```

Comment:
It was removed not in 'ARM' branch and latest changes, as (almost) all related to replication tasks had such conditions before:

```yaml
when:
    - specification.extensions.replication.enabled is defined
    - specification.extensions.replication.enabled
    - specification.extensions.replication.use_repmgr is defined
    - specification.extensions.replication.use_repmgr
```

Reply:
We may need a separate task for clean-up in develop since this is not directly related to ARM.

Reply:
New task is #2237. Changes were rolled back.